specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/schema/engine.schema.json

```diff
@@ -372,6 +372,14 @@
                         "type": "object",
                         "properties":
                         {
+                            "expect":
+                            {
+                                "type": "string",
+                                "enum":
+                                [
+                                    "json"
+                                ]
+                            },
```

Comment:
We need to put `validator` implementation-specific properties in the validator-specific schema patches so that engine doesn't need global knowledge of the possible properties used by all the validator implementations.

This currently includes
  - encoding
  - expect

Please refactor them out of the engine schema.